ci/unreal-engine.version

```diff
@@ -1,2 +1,2 @@
-dd42c1703769d1ed5510e0d56fb54646956d98d5
-59cf76706893d89f407e34518e8e7fbc90dd7e28
+HEAD 4.26-SpatialOSUnrealGDK-0.13.1-rc
```

Comment:
Should this be a hash?

Reply:
The release pipeline will make it a hash automatically during the final release